Sources/Tropos/Extensions/INInteraction.swift

```diff
@@ -0,0 +1,16 @@
+import Intents
+import TroposCore
+
+@available(iOS 12.0, *)
+extension INInteraction {
+    @objc(tr_checkWeatherInteraction)
+    static var checkWeather: INInteraction {
+        let intent = CheckWeatherIntent()
+        intent.suggestedInvocationPhrase = checkWeatherSuggestedPhrase
+        return INInteraction(intent: intent, response: nil)
+    }
+
+    private static let checkWeatherSuggestedPhrase = NSLocalizedString(
+        "Check the weather",
```

Comment:
Is there any potential for this to collide with another weather app using the same phrase?